### DISCUSSION TITLE
Selecting a specific map from beatmap listing outside of difficulty range doesnt go to map

### DISCUSSION THREAD

**Discussion by N0Y0U2114** [ppy/osu#30992] [Ideas] [answered]
When you select a map from the beatmap listing (or anywhere else you can select maps), and it is outside of the currently set difficulty range, a random map is selected and no indication is given as to why it didnt go to map.

https://github.com/user-attachments/assets/d224ae98-d706-4d07-83b8-82832381f674

Clicking on said map should either go to the map or have a flyout that you need to set your difficulty range to something else to be able to view said map

Heres one way you could show that

![OutOfRangeMap](https://github.com/user-attachments/assets/874d60a4-c4c7-4f4a-b9ce-e579ace0003f)

**Comment by peppy** [comment]
Could you please share your logs? I can't make this happen:


https://github.com/user-attachments/assets/11b8ce58-13f2-4df5-8152-699ce91c1c5e

If anything, the only issue here is that the beatmap doesn't hide after deselecting it.

**Reply by Joehuu** [reply]
Reproducing this requires to be in main menu before pressing the go to beatmap button.

**Reply by Joehuu** [reply, marked as answer]
This is also already tracked as an issue in https://github.com/ppy/osu/issues/29143.
